utils/analytics.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	xscservices "github.com/jfrog/jfrog-client-go/xsc/services"
+	"os"
+	"strings"
+)
+
+const (
+	githubActionsCi  = "github-actions"
+	jenkinsCi        = "jenkins"
+	gitlabCi         = "gitlab"
+	azurePipelinesCi = "azure-pipelines"
+)
```

Comment:
you already have: `vcsutils.VcsProvider` you can use this enum instead. we don't want code related to specific vsc providers at Frogbot.

Reply:
there are some missing Ci providers (those that are not related to some vsc provider). I created new values for CI providers in Frogbot consts.go file